Dima/netflix-roulette/src/Pages/FilmPage/FilmPage.tsx

```diff
@@ -69,18 +66,26 @@ class FilmPage extends React.Component<PropsFromRouteAndRedux> {
             <Header />
             <Link to="/">
               <Button 
-              isActive = { true }
-              buttonContent = "Search" />
+                isActive = { true }
+                buttonContent = "Search" 
+              />
             </Link>
           </div>
           {currentFilm && <FilmInfo film = { currentFilm } />}
         </div>
         <AdditionalPanel 
           genre = { String(genre) }
         />
-        <Films 
-          films = { films }
-        />
+        <InfiniteScroll
+          pageStart={ films.length }
+          loadMore={ () => this.handlePagination(films.length + 10, true) }
+          hasMore={ true }
```

Comment:
Eventually movies will end and you will get an error, so you have to write a check here. Make sure that total > this.props.movies.length;